tests/DiffSharp.Tests/TestTensor.AvgPool.fs

```diff
@@ -1,358 +0,0 @@
-﻿// Copyright (c) 2016-     University of Oxford (Atilim Gunes Baydin <gunes@robots.ox.ac.uk>)
```

Comment:
What happened to these tests?

Reply:
OK they ended up in TestOp.AvgPool.fs

Reply:
Yes, I'm thinking if we adopt this naming scheme things will be tidier in the long term.